src/index.js

```diff
@@ -80,6 +81,12 @@ export default class MediaQuery extends React.Component {
       matches: this._mql.matches
     })
   }
+  
+  componentDidUpdate(_, prevState) {
```

Comment:
@d4rky-pl @contra IMO, we have more use cases with dispatching onUpdate inside componentDidMount, or we should have both onUpdate and preUpdate?